recipes/ldns/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
You should be able to drop the duplciated `example.c` if you redirect to the main test package https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/test_v1_package/CMakeLists.txt :)

Reply:
Thanks for the suggestion. Done.